subsys/net/lib/utils/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
We have network utilities already in `subsys/net/ip/utils.c` file, no need to create a new file for these.

Reply:
> We have network utilities already in subsys/net/ip/utils.c file, no need to create a new file for these.

In subsys/net/ip/utils.c, we have internal utilities for Zephyr IP stack. This PR introduces generic, POSIX-compliant helper functions, which in particular intended to be portable and buildable with other POSIX systems. To elaborate, I initially thought of putting them under `lib/netutils/`, but as a compromise put under existing `subsys/net/lib/`. If their intentions there are not clear, we can move them under lib/.